lib/main.dart

```diff
@@ -41,16 +42,18 @@ const String AUTHOR = 'Hitesh Kumar Saini <saini123hitesh@gmail.com>';
 const String LICENSE = 'GPL-3.0';
 
 Future<void> main(List<String> args) async {
+  await Hive.initFlutter();
+  var configurationBox = await Hive.openBox('configuration');

```

Comment:
Ideally, I would like to move all `Hive` related method calls entirely into `Configuration` class.
Current `Configuration` class is actually really helpful in giving good intellisense, and one doesn't mess up with wrong key/box strings. A single `update` method does it all.

I can do it in this PR itself or you are free to do it. I'm already happy with what you've done. 🙂